ibis/bigquery/compiler.py

```diff
@@ -124,6 +127,76 @@ def _array_length(translator, expr):
     )
 
 
+def _arbitrary(translator, expr):
+    arg, where = expr.op().args
+    if where is not None:
+        arg_formatted = translator.translate(where.ifelse(arg, ibis.NA))
+    else:
+        arg_formatted = translator.translate(arg)
+    return 'ANY_VALUE({})'.format(arg_formatted)
+
+
+def _percentile(translator, expr):
+    args = expr.op().args
+    (arg, percentile_rank) = map(translator.translate, args[:2])
+    value_type = args[2]
+    command = dict(cont='percentile_cont', disc='percentile_disc')[value_type]
```

Comment:
How about `'percentile_{}'.format(value_type)`?

Reply:
Let's leave this out of this PR, and I'm going to refactor `Quantile` to be an `AnalyticOp` so that we don't have to have both `Quantile` and `Percentile`.